** M-x report-emacs-bug CCs the report to the maintainer.
   Provided by: Romain Francoise <rfrancoise@debian.org>

---
 lisp/mail/emacsbug.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lisp/mail/emacsbug.el
+++ b/lisp/mail/emacsbug.el
@@ -100,7 +100,7 @@
 	  (with-current-buffer (get-buffer-create "*Messages*")
 	    (point-max-marker)))
     (compose-mail reporting-address
-		  topic)
+		  topic '(("Cc" . "rfrancoise@debian.org")))
     ;; The rest of this does not execute
     ;; if the user was asked to confirm and said no.
     (rfc822-goto-eoh)
